app/controllers/users/confirmations_controller.rb

```diff
@@ -112,7 +112,12 @@ def after_confirmation_path_for(resource)
     end
 
     def process_confirmed_user
-      analytics.track_event('Email changed and confirmed', @confirmable)
+      if @confirmable.previous_changes.key?('email')
+        analytics.track_event('Email change confirmed', @confirmable)
```

Comment:
Not sure really, since we for sure don't want to localise them.   In any case that is unrelated to this PR.